web_src/js/components/DiffCommitSelector.vue

```diff
@@ -0,0 +1,191 @@
+<template>
+  <div class="ui dropdown custom">
+    <button
+      class="ui basic button"
+      @click.stop="toggleMenu()" @keyup.enter="toggleMenu()"
```

Comment:
Not sure if `keyup.enter` suffices/ if we need it.
There should be a global JS selector that acts on `ui button` to redirect keypresses (both <kbd>Enter</kbd> and <kbd>Space</kbd>) to a button click.

Reply:
Thanks - that was a bug. I removed the keyup.enter and now keyup opens the selector as expected.